src/middleware/builtin.spec.ts

```diff
@@ -609,6 +630,30 @@ describe('matchEventType', () => {
     assert.isTrue(fakeNext.called);
   });
 
+  it('should detect valid RegExp requests with app_mention', async () => {
+    const fakeNext = sinon.fake();
+    await matchEventType(/app_mention|app_home_opened/)({
+      logger,
+      client,
+      next: fakeNext,
+      context: {},
+      ...buildArgs(),
+    });
+    assert.isTrue(fakeNext.called);
+  });
+
+  it('should detect valid RegExp requests with app_home_opened', async () => {
+    const fakeNext = sinon.fake();
+    await matchEventType(/app_mention|app_home_opened/)({
```

Comment:
what's the difference between this test case and the one above it?

Reply:
There's no big difference, I just wanted to cover tests for the regex(_/app_mention|app_home_opened/_) specific to _app_home_opened_. We can remove this test case if it’s unnecessary to have redundant tests.
.